### PR TITLE
Removal of imagePullSecret from docs by default

### PR DESCRIPTION
**Pull request by sabre1041**
Removal of imagePullSecret from docs by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,8 +112,7 @@ kind: QuayEcosystem
 metadata:
   name: example-quayecosystem
 spec:
-  quay:
-    imagePullSecretName: redhat-pull-secret
+  quay: {}
```
Comment (thomasmckay) [inline README.md:115]: What do you think about adding a yaml comment at the above all of the `redhat-pull-secret` lines that says, "Required only if 'quay' and 'clair-jwt' images require pull secrets"

Comment (thomasmckay) [reply README.md:115]: alternatively, could put two example blocks, one with and one without the imagePullSecretName

Comment (sabre1041) [reply README.md:115]: im hesitant as that would really clutter up the documentation.

Comment (thomasmckay) [reply README.md:115]: How about two example blocks just here, one with and one without the secret. The rest merely omit.

**Review by thomasmckay — APPROVED** [review]
LGTM

**Pull request merged**